changelogs/fragments/3218-linode-inventory-plugin-typo-fixes.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "linode inventory plugin - fixes typos in the unit tests and module documentation"
```

Comment:
Docs and test changes shouldn't have a changelog fragment. Can you please remove this?

Reply:
Sure thing, deleted.